docs/resources/aws_shield_subscription.md

```diff
@@ -0,0 +1,55 @@
+---
+title: About the aws_shield_subscription Resource
+platform: aws
+---
+
+# aws\_shield\_subscription
+
+Use the `aws_shield_subscription` InSpec audit resource to test properties of a shield subscription.
+
+## Syntax
+
+ An `aws_shield_subscription` resource block uses the parameter to return a shield subscription.
+
+    describe aws_shield_subscription do
+      it { should exist }
+    end
+
+
+#### Parameters
+
+This resource does not expect any parameters.
+
+See also the [AWS documentation on Shield Subscriptions](https://docs.aws.amazon.com/waf/latest/developerguide/shield-chapter.html).
+
+
+## Properties
+
+|Property                      | Description|
+| ---                          | --- |
+|auto\_renew                   | If ENABLED, the subscription will be automatically renewed at the end of the existing subscription period. AutoRenew can be either ENABLED or DISABLED. |
+|end\_time                     | The date and time your subscription will end. |
+|limits                        | Specifies how many protections of a given type you can create. |
```

Comment:
It could be nice to mention that this is an array of items in this format: https://docs.aws.amazon.com/waf/latest/DDOSAPIReference/API_Limit.html

Reply:
updated with some more clarification on limits format